src/uproot/models/RNTuple.py

```diff
@@ -257,6 +284,15 @@ def field_form(self, this_id, seen):
             recordlist = [self.field_form(i, seen) for i in newids]
             namelist = [frs[i].field_name for i in newids]
             return ak._v2.forms.RecordForm(recordlist, namelist, form_key="whatever")
+        elif sr == 3:
+            # mother of a variant, i.e. union
+            keyname = self.col_form(this_id)
+            newids = []
+            for i, fr in enumerate(frs):
+                if i not in seen and fr.parent_field_id == this_id:
+                    newids.append(i)
+            recordlist = [self.field_form(i, seen) for i in newids]
+            return ak._v2.forms.UnionForm("i8", "i64", recordlist, form_key=keyname)
         else:
             # everything should recursive above this branch
             raise (AssertionError("this should be unreachable"))
```

Comment:
No biggie, but Python doesn't need all of these parentheses.

```suggestion
            raise AssertionError("this should be unreachable")
```

To minimize noise, the fewer the better. I don't know why black didn't do this automatically. (It's supposed to keep the AST unchanged, and removing parentheses would keep the AST unchanged; it's an AST, not a parsing tree!)